From d84a4c3b26dd2bdb14e6403eadc2cae8cca2fcec Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Joy=20Serqui=C3=B1a?=
 <44146839+essjay05@users.noreply.github.com>
Date: Mon, 27 Jan 2025 23:18:07 -0800
Subject: [PATCH] docs: updates buttons with icons a11y recommendations
 (#30400)

Updates Buttons with icons documentation to recommend that any icon
button be a minimum of 48x48 to be fully accessible and easily
clickable on smaller screens.
---
 src/material/button/button.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/material/button/button.md b/src/material/button/button.md
index 6b51d9963fc0..4ed6d42af291 100644
--- a/src/material/button/button.md
+++ b/src/material/button/button.md
@@ -68,7 +68,7 @@ with any assistive technology your application supports.
 Buttons or links containing only icons (such as `mat-fab`, `mat-mini-fab`, and `mat-icon-button`)
 should be given a meaningful label via `aria-label` or `aria-labelledby`. [See the documentation
 for `MatIcon`](https://material.angular.io/components/icon) for more
-information on using icons in buttons.
+information on using icons in buttons. Additionally, to be fully accessible the icon should have a minimum touch-target of 48x48 to ensure that the icon is easily clickable particularly on mobile devices and small screens.
 
 #### Toggle buttons
 [See the documentation for `MatButtonToggle`](https://material.angular.io/components/button-toggle)
